guide/kong_integration_k8s.md

```diff
@@ -0,0 +1,228 @@
+# Integrate Kong with your Kubernetes OpenFaaS cluster
+
+[Kong](https://getkong.org) is an API gateway that provides features such as security, logging, and rate limiting. By putting this in front of OpenFaaS you can quickly get access to these things and a lot more via [the many other plugins written](https://getkong.org/plugins/) for it.
+
+Below is a demo of how you could use Kong as an authentication layer for OpenFaaS.
+
+# Bring up Kubernetes
+
```

Comment:
I think I'd make the same point I did with swarm - let's link to the documentation instead. Guide for "Deploy K8s".

Might be better to deploy without qinikube so that this can be followed on a cloud environment too.

Reply:
if you look at line 23 I am linking to the faas k8s guide.
and I mentioned that this demo uses minikube, and you can skip that section if you have it running already.